source/support/faq.rst

```diff
@@ -4,104 +4,107 @@
 FAQ
 ***
 
-What is the Embassy?
-====================
-
-The internet as we know it is organized into questioners, or clients, and answerers, or servers. When you open a mobile email app, say Gmail, the app (client) begins asking questions: "have I received new mail?", "what are my last 50 messages?", "what drafts am I in the midst of writing?", and so on. Your app's questions are sent to and heard by a Google-run server which then provides answers back to the client and are subsequently displayed to the screen.
+General
+=======
 
-The Embassy is exactly that: your very own "answerer", just like Google's, except managed simply and with ease by and for you alone.
+What is Start9Labs?
+-------------------
+Start9Labs is a company based in Denver, CO that builds the Embassy and EmbassyOS.
 
-In other words, it is a generalized private personal server capable of running all sorts of self hosted open source software. 
-
-When you see your credit card information on your banking app, your messages in your texting app, your passwords in your password management app, all of that information comes from somewhere in the cloud: some server run by some company somewhere on the planet. Who can see the data stored in that server? Who can edit it? It's not always clear, but the increasingly common practice of selling your data to advertisers and the high-profile cyber-security breaches of the last decade suggest a pessimistic outlook.
+What is the Embassy?
+--------------------
+The Embassy is a "shelf-top" computer built using a Raspberry Pi (relevant link) for hardware and running EmbassyOS software.
 
-One thing is for certain though: if you control your server, then you control your data. Your finances, your communications, all of it is actually yours -- and only yours -- with an Embassy.
+What is EmbassyOS?
+------------------
+EmbassyOS is a new kind of Operating System (OS).  It is built from the ground up starting with a stripped-down version of Raspbian Buster Lite (relevant link).  It handles all operations of the device, including managing the Marketplace, Services, and Service data.
 
+What are EmbassyOS Services?
+----------------------------
+A Service can be any piece of software added to the Marketplace.  All services are "self-hosted," (relevant link?) meaning that you are in complete control of your data.  This means you can run your own "cloud!"
```

Comment:
maybe link to managing or available services?

Reply:
Linked to both.  Thank you for the feedback :)